### PR TITLE
Adding locales and notifications

### PR DESCRIPTION
**Pull request by Flobbah**
Adding locals to everything: Notifications, Target and Progressbars.
Notifications added to picking and collecting stuff.
I also removed some doubled code...
(Removed prints)

### CODE REVIEW

File: server/lsd.lua [hunk 1]
```diff
@@ -20,10 +21,11 @@ AddEventHandler('ps-drugprocessing:processThionylChloride', function()
 		local src = source
 		local Player = QBCore.Functions.GetPlayer(src)
 
-		Player.Functions.AddItem("thionyl_chloride", 1)
```
Comment (MonkeyWhisper) [inline server/lsd.lua:23]: Okay

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]
Hi, 

Please review my comments :) I only have one concern. 

Thank you so much for this PR, it'll be great for locales!!

File: fxmanifest.lua [hunk 2]
```diff
@@ -10,8 +10,13 @@ this_is_a_map 'yes'
 
 version '1.0.0'
 
-server_scripts {
+shared_scripts {
 	'config.lua',
+	'@qb-core/shared/locale.lua',
+	'locales/de.lua'
```
Comment (MonkeyWhisper) [inline fxmanifest.lua:16]: Please, English as default language.

File: client/target.lua [hunk 3]
```diff
@@ -331,18 +331,7 @@ Citizen.CreateThread(function()
                 type = "client",
                 event = "ps-drugprocessing:client:hydrochloricacid",
                 icon = "fas fa-radiation",
-                label = "Pickup Hydrochloric Acid Barrels",
-            },
-        },
-        distance = 4.0
-    })
-        exports['qb-target']:AddTargetModel("mw_chemical_barrel", {
```
[the remark this answers is not in the document]
Comment (Flobbah) [reply client/target.lua:339]: Becaus the code is already there. I don't see any point in having the same code twice?

File: fxmanifest.lua (hunk 2 above)
Comment (Flobbah) [reply fxmanifest.lua:16]: Oh, yeah, I'll change that. Wait for me to come home.

File: client/target.lua (hunk 3 above)
[the remark this answers is not in the document]
Comment (Flobbah) [reply client/target.lua:339]: As you can see here: https://i.gyazo.com/91650b6dcf00700001a8a4ed4369c473.png

[the remark this answers is not in the document]
Comment (MonkeyWhisper) [reply client/target.lua:339]: You are right! My fault, I overlooked that. Thank you for replying so quickly!

**Review by MonkeyWhisper — APPROVED** [review]

**Comment by MonkeyWhisper** [comment]
Thanks a lot!

**Pull request merged**